cli/src/semdep/maven_version.py

```diff
@@ -22,18 +22,20 @@ class ParsedMavenVersion:
 
 
 def parse_maven_version(version: str) -> MavenVersion:
-    m = re.compile(r"(\d)\.(\d)(?:\.(\d))?(.*)").match(version)
+    m = re.compile(
+        r"(?P<major>\d+)\.(?P<minor>\d+)(?:\.(?P<incremental>\d+))?(?P<qualifier>.*)"
```

Comment:
```suggestion
        r"(?P<major>\d+)[.](?P<minor>\d+)(?:[.](?P<incremental>\d+))?(?P<qualifier>.*)"
```

just something I like to do for readability, no need to commit and restart your CI pipelines